.jenkins/common.groovy

```diff
@@ -59,6 +59,20 @@ def runTestCommand (platform, project) {
                 python ../../tests/vision_tests/runVisionTests.py --runvx_directory ./bin --hardware_mode CPU --num_frames 100
                 python ../../tests/vision_tests/runVisionTests.py --runvx_directory ./bin --hardware_mode GPU --num_frames 100
                 python ../../tests/neural_network_tests/runNeuralNetworkTests.py
+                export OPENVX_DIR=$(pwd)/.
```

Comment:
our CI runs on groovy which also treats $ as a special character. You'll need to add the escape character '/' in front of all the '$'. I can see the $ character on lines 62, 63, and 70

Reply:
Thanks @eidenyoshida I will add those now